create.go

```diff
@@ -72,6 +72,10 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	if pidFilePath == "" {
```

Comment:
I try not to do generalities like "the go way" or "unix way", that tends to be more dogmatic than useful :)

What I'm trying to say: check constraints (can be also worded as establish strong invariants) as early as possible: this could be either program inputs are library entry points. Then, in the rest of the code, these invariants are established and don't need to be checked again.

It sometimes useful to have the nil value (0, nil, "") a valid input, that's a different thing though. That will lead to internal checks for nil, but that's just because it's a valid input and we need to do something special for that case.